client/src/components/Feed/NestedComments.js

```diff
@@ -53,6 +48,7 @@ const NestedComments = ({
   const [showReply, setShowReply] = useState(false);
   const [editComment, setEditComment] = useState(false);
   const [editedComment, setEditedComment] = useState(comment.content);
+  const isAuthor = useSelector(selectIsAuthor);
```

Comment:
is this a new pattern for redux? as much as I know selector return an specific sub object of the global state (state managed by redux) but a function. What you can do is return de actor Id (`selectActorId`) and then use that value to do the comparison.

Reply:
I found something similar here - https://github.com/reduxjs/reselect/issues/140#issuecomment-227554080

But `selectActorId` will work well enough. I will make the change.